packages/main/src/plugin/extension/extension-loader.ts

```diff
@@ -590,8 +588,11 @@ export class ExtensionLoader implements AsyncDisposable {
     const sorted: AnalyzedExtension[] = [];
     const explored = new Map<string, boolean>();
 
-    // visit all unvisited nodes
-    analyzedExtensions.forEach(node => {
+    const criticalExtensionIds = ['podman-desktop.podman', 'podman-desktop.compose', 'podman-desktop.kubectl-cli'];
```

Comment:
const file? Also may open up a can of worms of what we define as critical vs non critical.